app/addons/config/components.js

```diff
@@ -248,13 +252,12 @@ class ConfigOptionTrash extends React.Component {
 
   render() {
     return (
-      <td className="text-center config-item-trash config-delete-value"
-        onClick={this.showModal.bind(this)}>
-        <i className="icon icon-trash"></i>
+      <td className="text-center config-item-trash config-delete-value">
+        <i className="icon icon-trash" onClick={this.showModal}></i>
```

Comment:
Any reason to move the onClick handler to <i> instead of <td>?

Reply:
@popojargo Yes. Having the onClick inside `<td>` was causing it to be fired if you clicked any where on the modal (even the gray background). Meaning, the modal couldn't be closed when clicking on Cancel or the X button because it trigged the event to show the modal again.